inc/ticket.class.php

```diff
@@ -43,11 +43,7 @@ public static function pre_item_update(CommonDBTM $item)
             !empty(array_filter(
                 $item->input['_actors']['assign'] ?? [],
                 fn ($actor) => $actor['itemtype'] == 'Group'
-            ))
-            && (
-                isset($item->input['_from_assignment'])
-                && $item->input['_from_assignment']
-            )
```

Comment:
Should be kept to prevent warning about ```undefined index``` (see https://github.com/pluginsGLPI/escalade/issues/182)